content/post/2018-06-05_badcomkillsgoodanalytics.md

```diff
@@ -0,0 +1,88 @@
+
+###### Show and Tell
+Use time during your team meeting to have a team member present something they’ve learned or done. I’d suggest a tempo where a person isn’t asked to present more than once every 4-6 weeks. Be explicit with the goals ahead of time. For example, it might be more valuable for everyone on the team to know what kind of output your Tableau experts can create or why the work was meaningful to a stakeholder, instead of knowing how to create it themselves. Clarifying the purpose helps focus the presentation and the attention of the audience.
+
+## Talk about the business
+I’ve bounced with excitement telling my boss of an idea for a predictive model assisting another business unit, only to be thanked and reminded of how insignificant the reward vs my time would be for this project. I had better luck when I found cross-department team members to become collaborators, and as I learned their concerns, I found support for projects that weren’t just interesting to me, but also valuable for the company.
+
+When Analytics teams meet to discuss projects, everyone should understand what drives revenue (and profit), which parts of the business drive the most / least, and what’s the focus/priority for top management. Analytics teams are staffed with curious people. Business context makes their curiosity more likely to end in in projects worth doing.
+
+#### Improving “talk about the business”
+
+###### Lunch with Analytics
+Invite someone from another department to lunch with the analytics team each month. I’d scale to a roughly 1:5 ratio of “them” vs analytics team members. Don’t make it more complicated than necessary. The result should be a team of data people who understand a slice of the company and know someone in that department they can return to if they need to understand more.
+
+###### Read Reports
+A team has told you, this metric matters so much that we want to review it every day/week/month. If you want to find ways to leverage the rest of the analytics tool box, you’ll want to start with knowing why these metrics matter so much and what they mean. If their purpose don’t make sense, talk with the stakeholders about it. (see also: [_“Reporting is a Gateway"_]( https://www.locallyoptimistic.com/post/reporting-is-a-gateway-drug/) )
+
+###### Pop Quiz Yourself
+In the course of brainstorming you’ll notice people say, “I wonder how often X happens?” Make a running list of these small trivia style business questions and cut out every week or so to find the answers. These should be quick hits, not complex questions. For instance: “what’s our conversion rate?”, “how much mobile traffic do we get?”, “what’s the gender makeup of our audience?”, and “what’s the distribution of categorical variables in this data set?”.
```

Comment:
Maybe add these KPIs / questions to a dashboard?!

Reply:
added